src/main/java/com/scmspain/MsFcTechTestApplication.java

```diff
@@ -4,7 +4,6 @@
 import com.scmspain.configuration.TweetConfiguration;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
```

Comment:
I'm curious about that... if I put the `@SpringBootApplication` annotation, the tests fail, ¿why is that?
Nevertheless, a unused import must be erased.